Dockerfile

```diff
@@ -105,6 +105,12 @@ FROM core as development
 # Switch back to the root user to install development dependencies
 USER root:root
 
+# Install psql
+RUN apt-get update && \
+    apt-get install -y \
+    postgresql-client && \
```

Comment:
shouldn't we install mysql as well since we support mysql as well? Or is it already present?
You can test using mysql by setting the environment variable:
DB_HOST=mysql